linera-core/src/client/mod.rs

```diff
@@ -1516,6 +1527,7 @@ where
                         .await
                 })
             },
+            None,
```

Comment:
Did you mean to use `self.options.grace_period`, or is this deliberately `None`?